### PR TITLE
Better support for floats 

### PR DESCRIPTION
**Pull request by rohan-deshpande**
### Added 

* The ability to set floating point `step` props for `DatNumber` 
* The ability to disable the slider for `DatNumber` even when `min` and `max` props are provided

### CODE REVIEW

**Comment by rohan-deshpande** [comment]
This should probably be a minor version bump I guess so should it go to `v2.1.0`. I'll update the PR if you agree @claus

File: src/components/DatNumber.js [hunk 1]
```diff
@@ -16,9 +16,11 @@ export default class DatNumber extends Component {
     path: PropTypes.string,
     label: PropTypes.string,
     labelWidth: PropTypes.number,
+    customLabelWidth: PropTypes.number,
```
Comment (claus) [inline src/components/DatNumber.js]: Is this used at all?

Comment (rohan-deshpande) [reply src/components/DatNumber.js]: Hmm, let me check out if I'm using it at all in my app

Comment (claus) [reply src/components/DatNumber.js]: It's only in the PropTypes, but is never used anywhere else in DatNumber on your branch

Comment (rohan-deshpande) [reply src/components/DatNumber.js]: Ohh okay if it's not used in the actual component then can definitely remove. I'll do it. Thanks!

Comment (rohan-deshpande) [reply src/components/DatNumber.js]: @claus removed

**Pull request merged**